pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -105,6 +105,12 @@ public void createTenant(
 
         try {
             NamedEntity.checkName(tenant);
+            List<String> tenants = globalZk().getChildren(path(POLICIES), false);
+            int maxTenants = pulsar().getConfiguration().getMaxTenants();
+            //No locks for precise control
```

Comment:
So we can have a race and see that this limit is not enforced.
probably this is not a big deal (and enforcing it would be very costly) but we should state it clearly at least in the description of this pull request